packages/terra-consumer-layout/src/Layout.scss

```diff
@@ -8,7 +8,8 @@
     background-size: cover;
     overflow-x: hidden;
 
-    @media screen and (max-width: $terra-medium-breakpoint) {
+    @media screen and (max-width: $terra-medium-breakpoint),
+      screen and (orientation: landscape) and (max-width: $terra-large-breakpoint) {
```

Comment:
it looks like desktop computers are landscaped orientation (I assumed they didn't have orientation). This means this isn't an effective target for mobile...

Reply:
I searched and the only way I can determine to target mobile browsers (the ones that don't support this) is by doing
```css
@supports (-webkit-overflow-scrolling: touch) {
  background-image: none;
}
```

This is actually effective enough that you can remove the medium breakpoint media query to apply it.

Reply:
using orientation with combination of max-width will limit this change for mobile devices only (except ipad pro).  `@supports (-webkit-overflow-scrolling: touch)` approach also works -a   alternate way to handle this and can only be tested on actual devices. So we decided to keep the combination of orientation and max width.